src/core/tests/unit/vcfbed/test_vcftools.py

```diff
@@ -0,0 +1,258 @@
+import json
+import os
+import subprocess
+from os.path import join as pjoin
+from pathlib import Path
+
+import pandas as pd
+import pytest
+from ugbio_core.vcfbed import vcftools
+
+
+@pytest.fixture
+def resources_dir():
+    return Path(__file__).parent.parent.parent / "resources"
+
+
+def test_bed_files_output(resources_dir):
+    # snp_fp testing
+    data = pd.read_hdf(pjoin(resources_dir, "BC10.chr1.h5"), key="concordance")
+    snp_fp = vcftools.FilterWrapper(data).get_snp().get_fp().get_df()
+    assert not snp_fp["indel"].any()
+    assert all(x == "fp" for x in snp_fp["classify"])
+
+    # snp_fn testing
+    snp_fn = vcftools.FilterWrapper(data).get_snp().get_fn().get_df()
+    assert not snp_fn["indel"].any()
+    assert all(
+        row["classify"] == "fn"
+        or (row["classify"] == "tp" and (row["filter"] == "LOW_SCORE") and (row["filter"] != "PASS"))
+        for index, row in snp_fn.iterrows()
+    )
+
+    # hmer
+    hmer_fn = vcftools.FilterWrapper(data).get_h_mer().get_df()
+    assert all(hmer_fn["indel"])
+    assert all(x > 0 for x in hmer_fn["hmer_indel_length"])
+    hmer_fn = vcftools.FilterWrapper(data).get_h_mer(val_start=1, val_end=1).get_df()
+    assert all(hmer_fn["indel"])
+    assert all(x == 1 for x in hmer_fn["hmer_indel_length"])
+    hmer_fn = vcftools.FilterWrapper(data).get_h_mer(val_start=3, val_end=10).get_df()
+    assert all(hmer_fn["indel"])
+    assert all(x >= 3 and x <= 10 for x in hmer_fn["hmer_indel_length"])
+
+    # hmer_fp testing
+    hmer_fp = vcftools.FilterWrapper(data).get_h_mer().get_fp().get_df()
+    assert all(hmer_fp["indel"])
+    assert all([x > 0 for x in hmer_fp["hmer_indel_length"]])  # noqa: C419
```

Comment:
same [C419](https://docs.astral.sh/ruff/rules/unnecessary-comprehension-in-call/) should be easy to fix